src/settings/index.ts

```diff
@@ -172,6 +173,16 @@ export type AceBaseServerSettings = Partial<{
      */
     https: AceBaseServerHttpsSettings;
 
+    /**
+     * Provide your own server for AceBase to use
+     */
+    server: Server;
+
+    /**
+     * Root for the AceBase routes
+     */
+    route: string;
```

Comment:
Consider renaming to `rootPath`

Reply:
I'm not sure it's a good idea to rename it here. With the name `rootPath` it's not clear whether it refers to a filesystem path or a web path. I think `route` works much better here. If you don't like it, maybe something like `baseURL`, `prefixURL` or `webRoot` is better?

Reply:
Btw, I think `rootPath` it's ok for internal use in `RouteInitEnvironment`, since you the developer know what it refers to. I for user facing API it's quite confusing.

Reply:
@Azarattum if `rootPath` is an accurate description for internal use, I think it should be for external use as well. I strongly believe that if 1 thing represents the same in 2 different places, they must have the same name.

Reply:
What do you think about the other options I've suggested? We might use one of them both internally and externally for consistency. If you absolutely sure about `rootPath` however, I will make it so.

Reply:
Changed to `rootPath`.